content/guides/end-to-end-testing/amazon-cognito-authentication.md

```diff
@@ -255,7 +419,7 @@ describe('Cognito', function () {
 })
 ```
 
-## Adapting an Amazon Cognito App for Testing
+#### Adapting an Amazon Cognito App for Testing
```

Comment:
```suggestion
## Adapting an Amazon Cognito App for Testing
```

Reply:
Is this required for both logins or only for using programatic login?  If the later, we should specify

Reply:
only for programmatic, which is why I changed the header values to fall under the programmatic section. We can keep those the same, but it does feel a bit strange to call that out in each subsection

Reply:
Fair, we had called it out above which is why I had originally thought it should be it's own section

Reply:
I updated the header in e14bf7170 so the navbar shows the adapting section as such:
<img width="262" alt="Screen Shot 2022-12-01 at 12 41 00 PM" src="https://user-images.githubusercontent.com/3980464/205123457-7bc5eb2a-a11f-4c3b-9446-7427a46c49b2.png">